Diana_Mokhnitskaya/Lesson_5/HW5_task3.py

```diff
@@ -0,0 +1,11 @@
+import re
+import json
+
+task_string = " name=Amanda=sssss&age=32&&salary=1500&currency=euro "
+split_task_string = str(re.split('=|&|sssss', task_string))
```

Comment:
firs of all you should do strip. to remove spaces at the beginning and end of a string